src/main/java/org/snomed/snowstorm/core/data/services/MultiSearchService.java

```diff
@@ -229,6 +288,42 @@ public Set<String> getAllPublishedVersionBranchPaths() {
 		return publishedVersionBranchPaths;
 	}
 	
+	public Set<String> getCurrentRefsetBranchPaths() {
+		List<Branch> allBranches = branchService.findAll();
+		Set<String> refsetBranchPaths = new HashSet<>();
+		Map<String,String> refsetToCurrentBranchPathMap = new HashMap<>();
+		//Identify the latest Refset edit branch for each refset
+		//For example:
+		//MAIN/SNOMEDCT/REFSETS/REFSET-48353439008-1676579557135/EDIT-1676579564596
```

Comment:
Does this work by each commit being made on a uniquely named sub-branch?   This is sort of how we work for authoring, although we keep the same path name and just create a new object.   So the previous one gets given an 'end' timestamp which is equal to the time stamp of the next object.    Any concerns here @kaicode ?

So can you have multiple edits that are all "live" at the same time?   What happens to the 'end' parameter on the branch objects during successive edits?

Reply:
Each refset that is worked on gets it's own refset specifc branch with a timestamp when the edit cycle is begun. Under that branch there is a timestamped edit branch created for every time the refset is placed into "In Edit" mode. This can occur multiple times during an edit cycle. We do this since we can't delete branches but still need a way to undo changes while retaining the ability to recover undone edits in the rare case a massive change list was mistakenly cancelled. Each refset specific branch will only ever have one active edit branch, which is the one with the latest timestamp. This process was reviewed by Rory.